cypress/integration/step_definitions/Ui/single_vanities.cy.js

```diff
@@ -31,3 +31,125 @@ When('User clicks to Single Vanities', () => {
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+//Scenario: Add to Cart Button & Your Cart page functionality test
+
+let singleVanitiesPage=null
+before(()=>{
+    singleVanitiesPage =new SingleVanitiesPage();
+})
+
+And ('User clicks to random add-to-cart button' , () =>{
+    singleVanitiesPage.addToCartRandomProduct() 
+})
+
+And('it should direct to Your Cart page', () => {
+
+    const expectedTitle = 'Your Shopping Cart'; 
+    cy.title().should('contain', expectedTitle);
```

Comment:
we have common function this. shared the link below.

Reply:
Done. @KubraSA